### PR TITLE
adding known issue create resource button

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/open-cluster-management/backlog/issues/12865

### CODE REVIEW

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -92,6 +92,15 @@ From the console and Visual Web Terminal, users are unable to search for values
 When you are logged in as `kubeadmin` and you click the *Log out* option in the drop-down menu, the console returns to the login screen, but a browser tab opens with a `/logout` URL.
 The page is blank and you can close the tab without impact to your console.
 
+[#create-resource-fails-in-console]
+=== Create resource fails in the console
+
+// 2.0.11:12865
+
+When you select the _Create resource_ button from the _Welcome page_, you receive a creation error.
```
Comment (cadawson) [inline release_notes/known_issues.adoc]: "After you select the *Create resource* button on the _Welcome page_, you might receive an error message that an error occurred when the resource was created." 

Alternative to creation error? 

Should we specify the ACM Welcome page, or is that unnecessary?

File: release_notes/known_issues.adoc [hunk 2]
```diff
@@ -92,6 +92,15 @@ From the console and Visual Web Terminal, users are unable to search for values
...
+
+To resolve this issue, refresh the page and try again.
```
Comment (cadawson) [inline release_notes/known_issues.adoc:102]: "To work around this issue, refresh the _Create resource_ page and try again."
+
*Important:" The information that you entered will be lost when you refresh the _Create resource_ page.

**Review by cadawson — CHANGES_REQUESTED** [review]
Thanks Mikela. I added some content based on how I read the issue, but I might be missing the mark. Take a look and see if it makes sense.

File: release_notes/known_issues.adoc (hunk 1 above)
Comment (dockerymick) [reply release_notes/known_issues.adoc]: It is necessary, the error occurs only from the Welcome page

Comment (cadawson) [reply release_notes/known_issues.adoc]: I meant should we specify the "ACM" welcome page, or is that overkill?

File: release_notes/known_issues.adoc (hunk 2 above)
Comment (cadawson) [reply release_notes/known_issues.adoc:102]: *Create resource* button should be bold, not italic.

**Review by cadawson — APPROVED** [review]
Looks good! Thanks!

**Comment by dockerymick** [comment]
Thank you @cadawson !!!

**Pull request merged**